src/ApiClient.php

```diff
@@ -36,4 +36,10 @@ public function account() {
     $request = $this->get('https://api.envato.com/v1/market/private/user/account.json');
     return new Response\Account($request);
   }
+
+  public function search( $query = [] ) {
+      // todo: Do we need to verify the query params are valid before passing them to the API?
```

Comment:
I'd expect to at least know at this point whether or not the fields I'm passing in were correct. Easier here to throw an `ArgumentError` than trying to work out why no search results came back.

Perhaps a new class type for `Validators` and we can check it there?